egs/librispeech/ASR/conv_transducer_stateless/convrnnt.py

```diff
@@ -0,0 +1,1214 @@
+
+        self.conv5 = ScaledConv2d(
+            64,
+            1,
+            kernel_size=1,
+            stride=1,
+            padding=0,
+            bias=bias,
+        )
+
+    def forward(
+        self,
+        x: torch.Tensor,
+        src_key_padding_mask: Optional[torch.Tensor] = None,
+    ) -> Tuple[torch.Tensor, torch.Tensor]:
+        """Compute convolution module.
+
+        Args:
+            x: Input tensor (#time, batch, channels).
+            src_key_padding_mask: the mask for src keys per batch (optional)
+
+        Returns:
+            torch.Tensor: Output tensor (#time, batch, channels).
+        """
+        x = x.permute(1, 2, 0)  # (#batch, channels, time).
+        if src_key_padding_mask is not None:
+            x = x.masked_fill(
+                src_key_padding_mask.unsqueeze(1).expand_as(x), 0.0
+            )
+        x = x.unsqueeze(1)
+
+        # Conv1
+        if self.causal and self.lorder > 0:
+            # Make conv causal by
+            # manualy padding self.lorder zeros to the left
+            x = nn.functional.pad(x, (self.lorder, 0), "constant", 0.0)
+        x = self.conv1(x)
+        x = self.deriv_balancer1(x)
+        x = self.activation(x)
+
+        # Conv2
+        if self.causal and self.lorder > 0:
+            # Make conv causal by
+            # manualy padding self.lorder zeros to the left
+            x = nn.functional.pad(x, (self.lorder, 0), "constant", 0.0)
+        x = self.conv2(x)
+        x = self.deriv_balancer2(x)
+        x = self.activation(x)
+
+        # Conv3
+        if self.causal and self.lorder > 0:
+            # Make conv causal by
+            # manualy padding self.lorder zeros to the left
+            x = nn.functional.pad(x, (self.lorder, 0), "constant", 0.0)
+        x = self.conv3(x)
+        x = self.deriv_balancer3(x)
+        x = self.activation(x)
+
+        # Conv4
+        if self.causal and self.lorder > 0:
+            # Make conv causal by
+            # manualy padding self.lorder zeros to the left
+            x = nn.functional.pad(x, (self.lorder, 0), "constant", 0.0)
+        x = self.conv4(x)
+        x = self.deriv_balancer4(x)
+        x = self.activation(x)
+        # Conv5
+        # (N, 64, C, T) -> (N, 1, C, T) -> (N, C, T)
+        x = self.conv5(x)
+        x = x.squeeze(1).permute(2, 0, 1)
+        return x
+
+
+class SEModule(nn.Module):
+    """SE Module as defined in original SE-Nets with a few additions
+    Modified from rwightman`s squeeze_excite.py
+    """
+
+    def __init__(
+        self,
+        channels,
+        rd_ratio=1.0 / 16,
+        rd_channels=None,
+        rd_divisor=8,
+        bias=True,
+    ):
+        super(SEModule, self).__init__()
+        if not rd_channels:
+            rd_channels = self.make_divisible(
+                channels * rd_ratio, rd_divisor, round_limit=0.0
+            )
+        self.fc1 = nn.Conv1d(channels, rd_channels, kernel_size=1, bias=bias)
```

Comment:
Why not use `nn.Linear` to replace `nn.Conv1d`?

Reply:
ok, I will fix it.